server/datastore/mysql/migrations/tables/20220608113128_AddTransparencyURLToAppConfig.go

```diff
@@ -0,0 +1,51 @@
+package tables
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220608113128, Down_20220608113128)
+}
+
+func Up_20220608113128(tx *sql.Tx) error {
+	var raw []byte
+	row := tx.QueryRow(`SELECT json_value FROM app_config_json LIMIT 1`)
+	if err := row.Scan(&raw); err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return nil
+		}
+		return errors.Wrap(err, "select app_config_json")
+	}
+
+	var config fleet.AppConfig
+	if err := json.Unmarshal(raw, &config); err != nil {
+		return errors.Wrap(err, "unmarshal appconfig")
+	}
+
+	if config.FleetDesktop.TransparencyURL != "" {
+		return errors.New("unexpected transparency_url value in appconfig")
+	}
+
+	config.FleetDesktop.TransparencyURL = "https://fleetdm.com/transparency"
```

Comment:
Should we set empty as default? (and return the hardcoded value when the field is empty).

Thinking the case if we need to update the default URL (e.g. we change domain or path).